apps/playwright/.env.example

```diff
@@ -2,3 +2,8 @@
 # See: https://playwright.dev/docs/test-timeouts
 #TEST_TIMEOUT=30000
 #EXPECT_TIMEOUT=5000
+#BASE_URL=http://localhost:8000
+#ADMIN_USER_NAME=""
```

Comment:
this is just my idea .. not using this in this PR .. separated fixing tests into another one as I'm focused to get it running in Azure in this PR. Open to ideas on storing the diff credentials.